src/main/rx/cc2500_frsky_common.h

```diff
@@ -24,13 +24,21 @@
 
 #include "rx/rx_spi.h"
 
+typedef enum {
+  FRSKY_SPI_A1_SOURCE_VBAT = 0,
+  FRSKY_SPI_A1_SOURCE_EXTADC,
+  FRSKY_SPI_A1_SOURCE_CONST
+} frSkySpiA1Source_e;
+
+
 typedef struct rxFrSkySpiConfig_s {
-    uint8_t autoBind;
-    uint8_t bindTxId[2];
-    int8_t  bindOffset;
-    uint8_t bindHopData[50];
-    uint8_t rxNum;
-    uint8_t useExternalAdc;
+    uint8_t               autoBind;
+    uint8_t               bindTxId[2];
+    int8_t                bindOffset;
+    uint8_t               bindHopData[50];
+    uint8_t               rxNum;
+    frSkySpiA1Source_e    a1Source;
+    uint8_t               a1Const;
```

Comment:
What value do you see in making this adjustable? If a pilot has to use a constant A1 value because of TX behaviour, they will just ignore the value, and not bother to set it to anything specific.
In fact, I've seen knock-off FrSky RX that do not have support for A1, and they seem to be sending what is interpreted as 5V on the TX, and this seems to be working for them, so maybe we should just hardcode for 5V (50 I assume).